torch/nn/attention/_flex_attention.py

```diff
@@ -39,11 +39,127 @@ def _identity(
     return score
 
 
+_DEFAULT_BLOCKSPARSE_SIZE = 128
+
+
+class _BlockSparseMask:
```

Comment:
were you able to make this a named tuple?

Reply:
I think probably yes, but I'd like to explore as a follow-up PR.